swig/cspyce_typemaps.i

```diff
@@ -654,29 +654,36 @@ void handle_invalid_array_shape_x2d(const char *symname, PyArrayObject *pyarr, i
 }
 %}
 
+%{
+/**
+ We allow the conversion of a long array to an int array.  All other unsafe conversions
+ are disallowed.
+*/
+inline int adjust_convert_flags(int typecode, PyObject *input, int flags) {
```

Comment:
When I compile under Linux, this function is undefined:

```
>>> import cspyce
Set environment variable 'CSPICE_DEVELOPMENT' to '1' to ignore this error
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/seti/oops/pds-cspyce/cspyce/__init__.py", line 136, in <module>
    raise err
  File "/seti/oops/pds-cspyce/cspyce/__init__.py", line 130, in <module>
    from .cspyce2 import *
  File "/seti/oops/pds-cspyce/cspyce/cspyce2.py", line 13, in <module>
    import cspyce.cspyce1 as cspyce1
  File "/seti/oops/pds-cspyce/cspyce/cspyce1.py", line 16, in <module>
    from cspyce import cspyce0
  File "/seti/oops/pds-cspyce/cspyce/cspyce0.py", line 13, in <module>
    from . import _cspyce0
ImportError: /seti/oops/pds-cspyce/cspyce/_cspyce0.cpython-311-x86_64-linux-gnu.so: undefined symbol: adjust_convert_flags
```

However, if I remove the "inline", it compiles and works fine including passing the tests.